textproc/qpdf/Portfile

```diff
@@ -14,13 +18,10 @@ long_description    QPDF is a command-line program that does structural, \
                     content-preserving transformations on PDF files.
 
 homepage            http://qpdf.sourceforge.net/
+#github.tarball_from releases
```

Comment:
Why the comment?

Reply:
To remind me to uncomment it and remove the sourceforge master_sites when the port is updated next time.